src/com/esotericsoftware/kryo/io/ByteBufferOutput.java

```diff
@@ -276,6 +276,14 @@ public void writeBytes (byte[] bytes, int offset, int count) throws KryoExceptio
 		}
 	}
 
+	public void writeBytesFromLong (long bytes, int count) {

```

Comment:
`writeLong(long bytes, int count)`? Should we also have `writeInt(int bytes, int count)`?

Reply:
I have renamed to `writeLong(long bytes, int count)` and added `writeInt(int bytes, int count)`. I wonder if those methods should stay where they are in the `Output` class, i.e. in the section marked as `// byte:`, or should I move them to sections `// long: ` and `// int:` correspondingly (and to the same in `Input` class)?